app.js

```diff
@@ -15,6 +15,7 @@ const defaultData = {
   duration: 0,
   progress: 0.001, // force progress element to start out empty
   playing: false,
+  'played_count': 0,
```

Comment:
I'm not a fan of mixing snake_case and camelCase, could we just make this playedCount?